src/Features/Clonable.php

```diff
@@ -117,8 +117,10 @@ private function cloneRelations(CommonDBTM $source, bool $history): void
             $override_input[$classname::getItemField($this->getType())] = $this->getID();
 
            // Force entity / recursivity based on cloned parent, with fallback on session values
-            $override_input['entities_id'] = $this->isEntityAssign() ? $this->getEntityID() : Session::getActiveEntity();
-            $override_input['is_recursive'] = $this->maybeRecursive() ? $this->isRecursive() : Session::getIsActiveEntityRecursive();
+            if ($classname !== 'Profile_User') {
```

Comment:
It seems that there is a static property for this (used only for `Profile_User`).
```suggestion
            if ($classname::$disableAutoEntityForwarding !== false) {
```